pkg/services/authz/rbac_settings.go

```diff
@@ -57,3 +57,16 @@ func readAuthzClientSettings(cfg *setting.Cfg) (*authzClientSettings, error) {
 
 	return s, nil
 }
+
+type RBACServerSettings struct {
```

Comment:
So we now have `authzClientSettings`, `RBACServerSettings` and `iamOptions`.
Any way to rationalize?

Reply:
Yeah I know...

`authzClientSettings` is internal to package so it does not matter but they way I would think about it.

`authzClient` is more generic, it can be used to talk to either zanzana or rbac.
`RBACServerSettings` is used because it is only for RBAC server
`iamOptions` is used for the api server where we internally run RBAC server.